### PR TITLE
Deprecate wasm-c-api on big-endian and fix rot13 test

### LINKED ISSUE #1972: Rot13 test fails on s390x
When running the unittests on an s390x Docker image, the Rot13 test fails.

Here is a link to a GitHub Actions workflow that shows this failure: https://github.com/alexreinking/wabt/runs/8052934337?check_suite_focus=true

### PR DESCRIPTION
**Pull request by SoniEx2**
This includes #1971 and fixes #1972

Rationale in https://github.com/WebAssembly/wabt/issues/1972#issuecomment-1782130770

### CODE REVIEW

File: scripts/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM debian:latest
+
+WORKDIR /ws
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=UTC
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
+    echo $TZ > /etc/timezone && \
```
Comment (sbc100) [inline scripts/Dockerfile:9]: Do you know why this TZ hacking is needed.. seems a little odd.

**Review by sbc100 — COMMENTED** [review]
If thats all it takes to support these platforms its seems reasonable to me.

File: scripts/Dockerfile (hunk 1 above)
Comment (SoniEx2) [reply scripts/Dockerfile:9]: no idea, ask #1971 ? this PR is based on that one.

**Comment by SoniEx2** [comment]
this should not have broken the main build... where's the typo? >.<

**Comment by SoniEx2** [comment]
found the typo XD

**Comment by SoniEx2** [comment]
yeah this is sadly blocked on WebAssembly/wasm-c-api#180...

**Comment by SoniEx2** [comment]
it's possible these were always broken, but, uh, anyone wanna bisect these remaining failures?

**Comment by SoniEx2** [comment]
perhaps cc @rathann about i386/i686? also @alexreinking do you know anything about using a cross-compiler with these? running the whole compiler under qemu seems unnecessarily slow...

**Comment by rathann** [comment]
> perhaps cc @rathann about i386/i686? also @alexreinking do you know anything about using a cross-compiler with these? running the whole compiler under qemu seems unnecessarily slow...

x86_64 builds of gcc can compile for i686 without qemu or cross-compilation.

**Pull request closed without merging**